packages/offix-offline/src/offline/LocalDirectiveFilterLink.ts

```diff
@@ -3,7 +3,7 @@ import { hasDirectives, removeDirectivesFromDocument } from "apollo-utilities";
 import { localDirectivesArray, MUTATION_QUEUE_LOGGER } from "../utils/Constants";
 import * as debug from "debug";
 
-export const logger = debug.default(MUTATION_QUEUE_LOGGER);
+const logger = debug.default(MUTATION_QUEUE_LOGGER);
```

Comment:
This was actually exporting debug type - Top level apps will not work without @types/debug :)